custom/enikshay/ucr/data_sources/adherence.json

```diff
@@ -0,0 +1,158 @@
+{
+    "domains": [
+        "enikshay-test",
+        "enikshay-test-test",
+        "enikshay",
+        "enikshay-test-2",
+        "enikshay-test-3",
+        "enikshay-nikshay-migration-test",
+        "enikshay-domain-copy-test",
+        "enikshay-aks-audit",
+        "np-migration-3"
+    ],
+    "server_environment": [
+      "softlayer"
+    ],
+    "config": {
+        "referenced_doc_type": "CommCareCase",
+        "backend_id": "ES",
+        "description": "",
+        "base_item_expression": {
+        },
+        "table_id": "adherence",
+        "display_name": "adherence",
+        "configured_filter": {
```

Comment:
I just realized that you could exclude the cases that don't have DOSE_KNOWN_INDICATORS from the data source itself. However, if you ever wanted to change this list, we would need to rebuild the data source.